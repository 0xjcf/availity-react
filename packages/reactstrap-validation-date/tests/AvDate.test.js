import React from 'react';
import { render } from '@testing-library/react';
import { AvForm } from 'availity-reactstrap-validation';
import { AvDate } from '..';

describe('AvDate', () => {
  test('should render', () => {
    const { container } = render(
      <AvForm>
        <AvDate
          min="Min Date (yyyy-mm-dd)"
          max="Max Date (yyyy-mm-dd)"
          type="date"
          name="standAlone"
          aria-label="stand-alone"
          datepicker
        />
      </AvForm>
    );

    expect(container).toMatchSnapshot();
  });
});
