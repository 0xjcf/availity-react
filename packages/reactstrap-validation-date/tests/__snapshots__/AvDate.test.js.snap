// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvDate should render 1`] = `
<div>
  <form
    action="#"
    class="av-valid"
    method="get"
    novalidate=""
  >
    <div
      class="input-group"
    >
      <input
        aria-label="stand-alone"
        class="is-untouched is-pristine av-valid form-control"
        id="standAlone"
        max="Max Date (yyyy-mm-dd)"
        min="Min Date (yyyy-mm-dd)"
        name="standAlone"
        placeholder="yyyy-mm-dd"
        type="date"
        value=""
      />
      <div
        class="input-group-append"
      >
        <button
          class="btn btn-light"
          id="standAlone-btn"
          style="z-index: auto;"
          type="button"
        >
          <i
            aria-label="calendar"
            class="icon icon-calendar undefined"
          />
          <span
            class="sr-only"
          >
            Toggle Calendar
          </span>
        </button>
      </div>
    </div>
  </form>
</div>
`;
